static/src/js/components/AdminRetrievalsMetrics/AdminRetrievalsMetrics.jsx

```diff
@@ -76,6 +77,7 @@ export const AdminRetrievalsMetrics = ({
           <TemporalSelectionDropdown
             onChangeQuery={onChangeQuery}
             allowRecurring={false}
+            onMetricsTemporalFilter={onMetricsTemporalFilter}
```

Comment:
The only issue with adding this here is that we might be polluting the metrics because if we (the admins use the temporalSelectionDropdown its going to be counted on GA. It would be a pretty small subset of the total usage though so if it adds significant complexity to not use this then perhaps our best coarse is to use it like this.
The only alternative way I could see this working in a simple manner here might be to pass a stub to the component but, that does seem pretty hacky
```suggestion
            onMetricsTemporalFilter={someFuncThatDoesNothing}
```

Reply:
I think we could either have something in the event tell us which kind of temporal is being set. Or we could just ignore the admin pages. 

If you want to ignore the admin pages I wouldn't use a stub function, but just make the callback not required in TemporalSelectionDropdown, and only call it if it exists

Reply:
I think ignoring the admin pages does make the most sense I remember we talked about doing that during the design review. Making the callback not required and only calling it if its there does make the most sense to me